test/controllers/home_controller_test.rb

```diff
@@ -0,0 +1,8 @@
+require "test_helper"
```

Comment:
since we don't use mini test for automated testing, we can remove all the mini test files. 

You can delete the test folder completely and then you can set the generator of tests to be rspec. https://stackoverflow.com/a/33900226